test/commands/trigger_test.exs

```diff
@@ -1,13 +1,15 @@
 defmodule Cog.Test.Commands.TriggerTest do
-  use Cog.CommandCase, command_module: Cog.Commands.Trigger
+  use Cog.CommandCase, command_tag: :trigger
```

Comment:
That kind of change seems like it'd go with the current work; the tests are changing to reflect the change in the commands.